app/views/profile/show.html.erb

```diff
@@ -1,95 +1,69 @@
 <%= page_title t('profile.title') %>
 
-<% if Rails.env.development? %>
-  <div class="govuk-phase-banner">
-    <p class="govuk-phase-banner__content">
-      <strong class="govuk-tag govuk-phase-banner__content__tag">
-        development
-      </strong>
-      <span class="govuk-phase-banner__text">
-        <%= t 'profile.banner' %>
-      </span>
-    </p>
-  </div>
-<% end %>
+<%= link_to t('common.back'), root_path, class: 'govuk-back-link' %>
 
 <h1 class="govuk-heading-l"><%= t 'profile.title' %></h1>
 
-<div>
-  <h2 class="govuk-heading-m"><%= t 'profile.welcome' %> <%= @user.full_name %></h2>
+<table class="govuk-table">  
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header govuk-visually-hidden" scope="col"><%= t 'profile.item' %></th>
+      <th class="govuk-table__header govuk-visually-hidden" scope="col"><%= t 'profile.details' %></th>
+      <th class="govuk-table__header govuk-visually-hidden" scope="col"><%= t 'profile.action' %></th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= t 'profile.name' %></td>
+      <td class="govuk-table__cell"><%= @user.full_name %></td>
+      <td class="govuk-table__cell"></td>
+    </tr>
+    <tr class="govuk-table__row">
```

Comment:
the table is not fully responsive when the email is long. can we make it break lines properly?
e.g.
<img width="482" alt="image" src="https://user-images.githubusercontent.com/30629434/66919246-e5651880-f018-11e9-943b-9ece6b423105.png">

Reply:
the email is meant to break on next line so the Change button is not off-screen

Reply:
Strange i dont have that issue locally, the change button displays on screen
<img width="500" alt="Screen Shot 2019-10-17 at 10 06 01" src="https://user-images.githubusercontent.com/24409958/66994819-d12b2500-f0c5-11e9-8676-d8d61bd073c8.png">

Reply:
I guess your email is shorter, try changing it in the console to mine one and see.

Reply:
Yeah i see now, have pushed up a fix